.github/workflows/main.yml

```diff
@@ -27,17 +27,11 @@ jobs:
       - name: codegen test
         run: ./testbox codgen
   e2e_test:
-    runs-on: ubuntu-18.04
+    runs-on: self-hosted
     steps:
       - uses: actions/checkout@v2
-      - name: minikube start
-        run: |
-          ./testbox minikube download
-          ./testbox minikube start
-          ./testbox rook_install
       - name: build
         run: |
-          ./testbox operator-sdk_install
           docker login -u=tmaxanc+robot -p=${{ secrets.QUAY_PASSWORD }} quay.io
```

Comment:
need docker login?

Reply:
Yeap, docker login is needed before executing `operator-sdk build quay.io/tmaxanc/kubevirt-image-service:canary` which is the first command of `./testbox build`. 

As far as I know, docker login is on this workflow yaml file instead of testbox is to retrieve quay password without revealing it to the public.

Reply:
Just for future reference docker login session duration is unsure, so we'll keep it.